docs/kubernetes/windows.md

```diff
@@ -297,7 +297,127 @@ spec:
+
+
+### Known problems
+
+#### Packets from Windows pods are dropped
+Affects: Windows Server version 1803 (10.0.17134.1)
+Issues: https://github.com/Azure/acs-engine/issues/3037 
+
+There is a problem with the “L2Tunnel” networking mode not forwarding packets correctly specific to Windows Server version 1803. Windows Server version 1709 is not affected.
+
+Workarounds:
+**Fixes are still in development.** A Windows hotfix is needed, and willbe deployed by ACS-Engine once it's ready. The hotfix will be removed later when it's in a future cumulative rollup.
+
+
+#### Pods cannot resolve public DNS names
+
+Affects: Some builds of Azure CNI
+
+Issues: https://github.com/Azure/azure-container-networking/issues/147
+
+Run `ipconfig /all` in a pod, and check that the first DNS server listed is within your cluster IP range (10.x.x.x). If it's not listed, or not the first in the list, then an azure-cni update is needed.
+
+Workaround:
+
+1.	Get the kube-dns service IP with `kubectl get svc -n kube-system kube-dns`
+2.  Cordon & drain the node
+3.	Modify `C:\k\azurecni\netconf\10-azure.conflist` and make it the first entry under Nameservers
+4.  Uncordon the node
+
+Example:
+```
+{
+    "cniVersion":  "0.3.0",
+    "name":  "azure",
+    "plugins":  [
+                    {
+                        "type":  "azure-vnet",
+                        "mode":  "tunnel",
+                        "bridge":  "azure0",
+                        "ipam":  {
+                                     "type":  "azure-vnet-ipam"
+                                 },
+                        "dns":  {
+                                    "Nameservers":  [
+                                                        "10.0.0.10",
+                                                        "168.63.129.16"
+                                                    ],
+                                    "Search":  [
+                                                   "default.svc.cluster.local"
+                                               ]
+                                },
+…
+```
+
+#### Pods cannot resolve cluster DNS names
+
+Affects: Azure CNI plugin <= 0.3.0
+
+Issues: https://github.com/Azure/azure-container-networking/issues/146
+
+If you can't resolve internal service names within the same namespace, run `ipconfig /all` in a pod, and check that the DNS Suffix Search List matches the form `<namespace>.svc.cluster.local`. An Azure CNI update is needed to set the right DNS suffix.
+
+Workaround:
+1.	Use the FQDN in DNS lookups such as `kubernetes.kube-system.svc.cluster.local`
+2.	Instead of DNS, use environment variables `* _SERVICE_HOST` and `*_SERVICE_PORT` to find service IPs and ports in the same namespace
+
+
+#### Pods cannot ping default route or internet IPs
+
+Affects: All acs-engine deployed clusters
+
+ICMP traffic is not routed between private Azure vNETs or to the internet.
+
+Workaround: test network connections with another protocol (TCP/UDP)

```

Comment:
Should we clarify how? Like give an example such as:
The following will *not* work to demonstrate connectivity:
`ping 123.456.78`
Instead, you should use something like:
use `curl -Uri http://123.456.78 -UseBasicParsing`